site/docs/guidelines/typography.mdx

```diff
@@ -30,6 +30,48 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
 * Text color / contrast is crucial. Ensure color ratios comply with WCAG 2.1 AA. See [Color guidelines](/guidelines/color) for text color guidelines.
 * White space is equally important. Let your text breathe.
 
+### Headings
+
+- **Display 0:** Use for brand moments including, introductory screens (Onboarding, Success pages etc.) or landing pages (Log In and Capture Intro etc.).
+
+- **Heading 1:** H1 is used for page Titles in the [Title Block](/guidelines/header/#title-block) of the page [Header](/guidelines/header/)
```

Comment:
Bit of a nit, but I'd prefer not to mix the language here. H1 could be confused with the H1 HTML tag, when we're actually referring to the "heading-1" variant of Heading component (in the component library) or Headings / Heading 1 (in Figma).